crates/solver/src/settlement_submission/flashbots_settlement.rs

```diff
@@ -249,19 +251,19 @@ impl<'a> FlashbotsSolutionSubmitter<'a> {
             if let Err(err) = method.clone().view().call().await {
                 if let Some((_, previous_tx)) = previous_tx.as_ref() {
                     if let Err(err) = self.flashbots_api.cancel(previous_tx).await {
-                        tracing::error!("flashbots cancellation request not sent: {:?}", err);
+                        tracing::debug!("flashbots cancellation request not sent: {:?}", err);
```

Comment:
nit: maybe still warn since we want to get rid of those message by deserialising the cancellation response properly, right?

Reply:
I'd also like it as `info` or `warn`.

Reply:
Currently, valid cancellation result containing false also raises error. Do we still want to trace with warn even if this case happens, or should we change the return type of `cancel` from `Result<()>` to `Result<bool>` and handle the success outside?

Reply:
Set to warn.